test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/Common/Queue/IQueueManager.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.Common
+{
+    // Interface for Management of External Resources required for testing.
+    public interface IQueueManager<Request, Response>
+    {
```

Comment:
I see that the concrete implementation of this interface do not have the requirement of implmenting all these methods. Is that the two kinds of requirements by the Azure Queue infrastructure. Think if it can be created in a way that the two implementations can get only the methods it needs to implement or if the left methods will come in future, that is Ok.

Reply:
Yes. We plan to add it in the coming future.